addons/binding/org.openhab.binding.facerecognition/.classpath

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Could you please migrate to the new buildsystem? Have a look at the sticky issue for more information.